packages/handlersjs-http/lib/handlers/error.handler.spec.ts

```diff
@@ -0,0 +1,179 @@
+import { HttpHandlerResponse } from '../models/http-handler-response';
+import { ErrorHandler } from './error.handler';
+
+const response: HttpHandlerResponse = {
+  body: 'upstream response body',
+  headers: {
+    location: `http://test.be`,
+  },
+  status: 400,
+};
+
+const response200: HttpHandlerResponse = {
+  body: 'upstream successful response body',
+  headers: {
+    location: `http://test.be`,
+  },
+  status: 200,
+};
+
+describe('error_handler', () => {
+
+  const errorHandlerTrue = new ErrorHandler(true);
+  const errorHandlerFalse = new ErrorHandler();
+
+  it('should be instantiated correctly', () => {
+
+    expect(errorHandlerTrue).toBeTruthy();
+
+  });
+
+  describe('handler', () => {
```

Comment:
A lot of these tests can also be done by a `it.each` block.

Reply:
I'll check that out. Thanks for the tip.